src/workers/notifications/index.ts

```diff
@@ -0,0 +1,55 @@
+import { NotificationWorker } from './worker';
+import { Worker } from '../worker';
+import {
+  generateNotification,
+  storeNotificationBundle,
+} from '../../notifications';
+import communityPicksFailed from './communityPicksFailed';
+import communityPicksSucceeded from './communityPicksSucceeded';
+import communityPicksGranted from './communityPicksGranted';
+import articlePicked from './articlePicked';
+import articleNewCommentPostCommented from './articleNewCommentPostCommented';
+import articleNewCommentCommentCommented from './articleNewCommentCommentCommented';
+import articleUpvoteMilestone from './articleUpvoteMilestone';
+import articleReportApproved from './articleReportApproved';
+import articleAnalytics from './articleAnalytics';
+import sourceRequest from './sourceRequest';
+import commentMention from './commentMention';
+import commentReply from './commentReply';
+import commentUpvoteMilestone from './commentUpvoteMilestone';
+
+function notificationWorkerToWorker(worker: NotificationWorker): Worker {
+  return {
+    ...worker,
+    handler: async (message, con, logger) => {
```

Comment:
Ah, so this would execute after the original handler, so does it become one bg worker?

Reply:
This will be executed by the background worker. Think of it as a background worker for a specific use case of generating notifications. It has very structured workflow so I thought it may be nice to have it

Reply:
Yeah agreed, it's a nice flow and quite dynamic. 🥳